web/profiles/custom/yalesites_profile/config/sync/core.entity_form_display.paragraph.basic_view.default.yml

```diff
@@ -0,0 +1,23 @@
+uuid: 68006630-adbd-42e5-a66e-73e8bfdfa8bc
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.paragraph.basic_view.field_vb_params
+    - paragraphs.paragraphs_type.basic_view
+  module:
+    - ys_views_basic
+id: paragraph.basic_view.default
+targetEntityType: paragraph
+bundle: basic_view
+mode: default
+content:
+  field_vb_params:
```

Comment:
Please add the all the new things to the data model. I think this is just the paragraphs and paragraph fields tab at this time. [Data Model](https://yaleedu.sharepoint.com/:x:/r/sites/YaleSitesUpgradeProgram/_layouts/15/Doc.aspx?sourcedoc=%7B2B574C06-BAC3-41E2-924A-F4562054F849%7D&file=Proposed%20Content%20Model.xlsx&action=default&mobileredirect=true&wdOrigin=TEAMS-ELECTRON.teams.files&wdExp=TEAMS-CONTROL&wdhostclicktime=1644421397192&cid=1b5ea67e-ee90-4fbb-b504-f4dc05b2d87b)

Reply:
Added to data model.